assignments/objects.js

```diff
@@ -19,27 +19,78 @@ const example = {
 
 // Write your intern objects here:
 
-
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F",
+};
+
+const intern2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function (banana) {
+    return `Hello, my name is  ${banana}` 
+  
+  }
+};
+console.log(intern2.speak("Kennan"))
+
+const intern3 = {
+  "id": 3,
+  "name": "Keven",
+  "email": "mummery2@wikimedia.org",
+  "gender": "M",
+};
+
+const intern4 = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M",
+};
+
+const intern5 = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+};
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(intern1.name)
 // Kennan's ID
-
+console.log(intern2.id)
 // Keven's email
-
+console.log(intern3.email)
 // Gannie's name
-
+console.log(intern4.name)
 // Antonietta's Gender
+console.log(intern5.gender)
+
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+console.log(kennan.speak("Hello, my name is " + kennan.name + "!"));
```

Comment:
move the function inside of your object, and just console log `console.log(kennan.speak())`